pkgs/applications/audio/pavucontrol/default.nix

```diff
@@ -10,6 +10,14 @@ stdenv.mkDerivation rec {
     sha256 = "1qhlkl3g8d7h72xjskii3g1l7la2cavwp69909pzmbi2jyn5pi4g";
   };
 
+  patches = [
+    # Can be removed with the next version bump
+    (fetchpatch {
+      name = "streamwidget-fix-drop-down-wayland.patch";
+      url = "https://gitlab.freedesktop.org/pulseaudio/pavucontrol/-/merge_requests/20.patch";
```

Comment:
Please always use some URLs containing the commit IDs, like `https://gitlab.freedesktop.org/pulseaudio/pavucontrol/-/merge_requests/20/diffs?commit_id=ae278b8643cf1089f66df18713c8154208d9a505.patch` in this case.

The MR might be closed already, so it wouldn't change for this patch, but this will be a bad example of a patch url in nixpkgs.

Reply:
> Please always use some URLs containing the commit IDs, like `https://gitlab.freedesktop.org/pulseaudio/pavucontrol/-/merge_requests/20/diffs?commit_id=ae278b8643cf1089f66df18713c8154208d9a505.patch` in this case.

This link does not contain a valid patch, I'll just use the commit on master instead: `https://gitlab.freedesktop.org/pulseaudio/pavucontrol/-/commit/ae278b8643cf1089f66df18713c8154208d9a505.patch`.

Reply:
Done.

Reply:
Makes sense - but add the link to the MR in the comment above.